horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
```

Comment:
Yes, this example break the rule: E203 whitespace before ‘:’ and E241 multiple spaces after ':'.
To make it conform, it look like this:
```
            BUILDING_PURPOSE.ROAD: (30, 30, 30),
            BUILDING_PURPOSE.FISHER: (128, 128, 128),
            BUILDING_PURPOSE.LUMBERJACK: (30, 255, 30),
            BUILDING_PURPOSE.TREE: (0, 255, 0),
            BUILDING_PURPOSE.FARM: (128, 0, 255),
            BUILDING_PURPOSE.POTATO_FIELD: (255, 0, 128),
            BUILDING_PURPOSE.PASTURE: (0, 192, 0),
            BUILDING_PURPOSE.WEAVER: (0, 64, 64),
            BUILDING_PURPOSE.SUGARCANE_FIELD: (192, 192, 0),
```